scripts/shared/lib/clusters_kind

```diff
@@ -243,20 +243,16 @@ function provider_failed() {
 function prepare_ovn() {
     export OVN_IMAGE="localhost:5000/ovn-daemonset-f:latest"
 
-    echo "Building ovn-kubernetes with interconnect (OVN-IC) from source"
-    echo "This will become unnecessary if OVN CI image publishing is fixed"
-    echo "https://github.com/ovn-org/ovn-kubernetes/actions/workflows/docker.yml"
     rm -rf ovn-kubernetes
     git clone https://github.com/ovn-org/ovn-kubernetes
     pushd ovn-kubernetes || exit
-    git checkout d462cf088c44ff501e5fcf746e9163193afa2dc4
 
-    make -C go-controller
+    # When updating commit, Update the OVN_SRC_IMAGE to the corressponding commit
```

Comment:
Do we still need to clone the ovn-kubernetes repository?

Reply:
Oh, for the `kind.sh` script.

Reply:
ya that is right.